airflow/www_rbac/templates/airflow/dag.html

```diff
@@ -21,8 +21,7 @@
 
 {% block head_css %}
   {{ super() }}
-  <link href="{{ url_for('static', filename='main.css') }}" rel="stylesheet">
```

Comment:
What happened to this?

Reply:
This css has already been included in the base file i.e., `appbuilder/baselayout.html` 

**structure:** dag.html `extends > ` airflow/master.html `extends >` appbuilder/baselayout.html